test/parallel/test-assert-objects.js

```diff
@@ -97,6 +97,46 @@ describe('Object Comparison Tests', () => {
           actual: [1, 'two', true],
           expected: [1, 'two', false],
         },
+        {
+          description: 'throws when comparing [0] with [-0]',
+          actual: [0],
+          expected: [-0],
+        },
+        {
+          description: 'throws when comparing [0, 0, 0] with [0, -0]',
+          actual: [0, 0, 0],
+          expected: [0, -0],
+        },
+        {
+          description: 'throws when comparing [-0] with [0]',
+          actual: [0],
+          expected: [-0],
+        },
```

Comment:
```suggestion
```

Reply:
@LiviaMedeiros how come? Why do you think I should remove this?

Reply:
It’s a duplicate, look two test cases up

Reply:
Ops, I was checking on the phone, that was tough to spot 😅 the test doesn't have to be removed, the description is right but the test case is wrong. I will fix when home 😊